tests/e2e/multicluster/multicluster_multiprimary_test.go

```diff
@@ -253,7 +253,8 @@ spec:
 
 				AfterAll(func(ctx SpecContext) {
 					if CurrentSpecReport().Failed() {
-						common.LogDebugInfo()
+						common.LogDebugInfo(k1)
```

Comment:
One small improvement could be that while creating the [kubectl instance](https://github.com/istio-ecosystem/sail-operator/blob/16c3e03e691f6aa5b5f036beebcd2b61154d4baf/tests/e2e/util/kubectl/kubectl.go#L34), we can pass the cluster name and log the cluster-name as part of debug info. This would be useful for the multicluster tests.

Reply:
Added.

Thanks for the help with the proper implementation.